hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/KafkaAuthorizationProperties.java

```diff
@@ -47,4 +49,20 @@ public String getPassword() {
     public void setPassword(String password) {
         this.password = password;
     }
+
```

Comment:
I know your code does not change name of the class, but this class should be actually named accordingly to the `authentication` not `authorization`, e.g. `KafkaAuthenticationProperties`. The same holds for all places where the properties are used.

Reply:
I cleaned up a bit things related to Kafka auth, added fallback setters to properties classes to ensure backward compatibility. Please check last two commits.